modules/aws-iam-policies/main.tf

```diff
@@ -96,9 +95,9 @@ resource "aws_iam_role_policy_attachment" "emr_creator_policy_attachment" {
   policy_arn = aws_iam_policy.emr_creator_minimal_policy.arn
 }
 
-// IAM role policy attachment(s) that attach s3 policy ARNs to Tamr user IAM role
-resource "aws_iam_role_policy_attachment" "emrfs_user_s3_policy" {
-  count      = length(var.s3_policy_arns)
+// IAM role policy attachment(s) that attach additional policy ARNs to Tamr user IAM role
+resource "aws_iam_role_policy_attachment" "additional_user_policies" {
+  count      = length(var.additional_policy_arns)
   role       = var.aws_role_name
-  policy_arn = element(var.s3_policy_arns, count.index)
+  policy_arn = element(var.additional_policy_arns, count.index)
```

Comment:
This is a comment for a later iteration, not regarding this PR specifically. But I prefer having 
```
resource "aws_iam_role_policy_attachment" "additional_user_policies" {
  for_each   = toset(var.additional_policy_arns)
  role       = var.aws_role_name
  policy_arn = each.value
```
instead of using `count`.

Reason being: using `count`, in the state we'd have the multiple resources like `aws_iam_role_policy_attachment.additional_user_policies[0]`, `aws_iam_role_policy_attachment.additional_user_policies[1]`, 
. . . 
`aws_iam_role_policy_attachment.additional_user_policies[n]`.
If the order of the list is updated, terraform WILL "RECREATE" the infrastructure since state specification is'nt the same (`resource.id[0]` wouldn't be the same as before).
Instead, if you have `for_each` using a `set`, the key of each resource would be the ARN itself... something like this:
`aws_iam_role_policy_attachment.additional_user_policies["arn:aws:iam::policy/blabla"]`, `aws_iam_role_policy_attachment.additional_user_policies["arn:aws:iam::policy/bleble"]`, 
. . . 
`aws_iam_role_policy_attachment.additional_user_policies["arn:aws:iam::policy/blublu"]`.

Having it this way the order of the values in the input list won't matter, for the state file is being recording with its ARN value, not its position in the list.